service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/C21OrderBundle.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class C21OrderBundle {
+    private final DocumentReference c21OrderDocument;
+    private final String orderTitle;
+    private final String orderDate;
+    private final String judgeTitleAndName;
+}
```

Comment:
I am really surprised there isn't a List<Element<C21Order>> in here 🤔

Reply:
we could do that - although, the AC doesn't actually care about displaying the orderDetails in the tab, only on the document. if we do list element then we store the whole c21Order object including the details, which are likely to be long - just a database consideration, mentioned this to adam too. still, I'll refactor to use list element as its a lot neater

Reply:
well in ccd a collection of a complex type is List<Element<the complex type>> in java.

Reply:
yep I understand this better now, thanks! I'd like to do this change (see comment above though)